### DISCUSSION TITLE
Content Accordion / Collapse

### DISCUSSION THREAD

**Discussion by matbrgz** [outline/rich-markdown-editor#413] [Ideas] [answered]
Hello,

Notion has an awesome feature that is collapse content.

![image](https://user-images.githubusercontent.com/23284276/111851052-b1b6ae00-88f0-11eb-909a-259bc8c324bf.png)
![image](https://user-images.githubusercontent.com/23284276/111851045-ae232700-88f0-11eb-8b74-1c977dde1fdc.png)

**Comment by tommoor** [comment, marked as answer]
The natural place to start with collapsing is probably headings:

https://github.com/outline/rich-markdown-editor/issues/169

**Reply by matbrgz** [reply]
Works for me, closing from here;
